packaging/test_plugin_event_handler-data_object_modified.py

```diff
@@ -61,23 +61,23 @@ def event_handler_configured(arg=None):
            }
         )
 
-        irods_config.commit(irods_config.server_config, irods_config.server_config_path)
-
-        IrodsController().restart()
 
-        try:
+    try:
+        with lib.file_backed_up(filename):
+            irods_config.commit(irods_config.server_config, irods_config.server_config_path)
+            IrodsController().reload_configuration()
             yield
-        finally:
-            pass
+    finally:
+        IrodsController().reload_configuration()
 
 @contextlib.contextmanager
 def event_handler_fail_policy_configured(arg=None):
     filename = paths.server_config_path()
-    with lib.file_backed_up(filename):
-        irods_config = IrodsConfig()
-        irods_config.server_config['advanced_settings']['rule_engine_server_sleep_time_in_seconds'] = 1
 
-        irods_config.server_config['plugin_configuration']['rule_engines'].insert(0,
+    irods_config = IrodsConfig()
+    irods_config.server_config['advanced_settings']['rule_engine_server_sleep_time_in_seconds'] = 1
```

Comment:
Hmm, is `rule_engine_server_sleep_time_in_seconds` an actual config property?

_Noticed other instances too._

Reply:
All tests passed. Resolving.